src/data/queries/mediaDocument.ts

```diff
@@ -0,0 +1,46 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { drupalClient } from '@/lib/drupal/drupalClient'
+import { MediaDocument } from '@/types/formatted/media'
+import { DrupalMediaDocument } from '@/types/drupal/media'
+
+// Define the query params for fetching node--media_document.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude(['document'])
```

Comment:
including `document` throws an error for me, `field_document` seems to be the actual field name